src/main/java/com/telefonica/iot/cygnus/interceptors/GroupingInterceptor.java

```diff
@@ -230,16 +306,29 @@ public void close() {
      * Builder class for this new interceptor.
      */
     public static class Builder implements Interceptor.Builder {
-        private String matchingTableFile;
+        private String groupingRulesFileName;
  
         @Override
         public void configure(Context context) {
-            matchingTableFile = context.getString("matching_table", null);
+            String groupingRulesFileNameTmp = context.getString("grouping_rules_conf_file");
+            String matchingTableFileName = context.getString("matching_table");
+            
+            if (groupingRulesFileNameTmp != null && groupingRulesFileNameTmp.length() > 0) {
+                groupingRulesFileName = groupingRulesFileNameTmp;
+                LOGGER.debug("[de] Reading configuration (grouping_rules_file=" + groupingRulesFileName + ")");
+            } else if (matchingTableFileName != null && matchingTableFileName.length() > 0) {
+                groupingRulesFileName = matchingTableFileName;
+                LOGGER.debug("[de] Reading configuration (matching_table=" + groupingRulesFileName + ")"
```

Comment:
Not in this case, because we are really reading a "matching_table" parameter :) This code tries to ensure backward compatibility. Nevertheless, please observe the " -- DEPRECATED, use grouping_rules_file instead" message.